Source/WebCore/accessibility/AXCoreObject.cpp

```diff
@@ -1540,8 +1540,8 @@ void AXCoreObject::appendRadioButtonGroupMembers(AccessibilityChildrenVector& li
 AXCoreObject* AXCoreObject::parentObjectUnignored() const
 {
     if (role() == AccessibilityRole::Row) {
-        if (auto* table = exposedTableAncestor())
-            return table;
+        if (RefPtr table = exposedTableAncestor())
+            return table.get();
```

Comment:
Another place where I don't understand why a smart pointer is necessary. All we do is null-check the raw pointer. If this _is_ actually necessary, I'm fully willing to accept this is my lack of understanding of safety, or alternatively willing to accept that the static analyzer is not very smart...just want to know which one it is 🙂